components/enterprise-contract/base/kustomization.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/enterprise-contract/core?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/enterprise-contract/policies?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
```

```suggestion
  - https://github.com/konflux-ci/konflux-ci/konflux-ci/enterprise-contract?ref=ff4de1214c9e8306baa54830d72284f7500dcd62
```
Refer to the root folder kustomization file which already contains all the resources